src/commands/link/link.ts

```diff
@@ -266,33 +249,33 @@ export const link = async (options: OptionValues, command: BaseCommand) => {
 
   // Site id is incorrect
   if (siteId && isEmpty(siteData)) {
-    log(`"${siteId}" was not found in your Netlify account.`)
-    log(`Please double check your siteID and which account you are logged into via \`netlify status\`.`)
-    return exit()
+    NetlifyLog.error(`"${siteId}" was not found in your Netlify account.`)
+    outro({
+      message: `Please double check your siteID and which account you are logged into via \`netlify status\`.`,
+    })
   }
 
   if (!isEmpty(siteInfo)) {
     // If already linked to site. exit and prompt for unlink
-    log(`Site already linked to "${siteData.name}"`)
-    log(`Admin url: ${siteData.admin_url}`)
-    log()
-    log(`To unlink this site, run: ${chalk.cyanBright('netlify unlink')}`)
+    NetlifyLog.info(`Site already linked to "${siteData.name}"`)
+    NetlifyLog.message(`Admin url: ${siteData.admin_url}`)
+    outro({ message: `To unlink this site, run: ${chalk.cyanBright('netlify unlink')}` })
   } else if (options.id) {
     try {
       siteData = await api.getSite({ site_id: options.id })
-    } catch (error_) {
-      // @ts-expect-error TS(2571) FIXME: Object is of type 'unknown'.
-      if (error_.status === 404) {
-        error(new Error(`Site id ${options.id} not found`))
+    } catch (error: unknown) {
+      if ((error as { status: number }).status === 404) {
+        NetlifyLog.error(new Error(`Site id ${options.id} not found`))
```

Comment:
Casing inconsistency

```suggestion
        NetlifyLog.error(new Error(`Site ID ${options.id} not found`))
```